### DISCUSSION TITLE
backup error :barmanObjectStore  unrecognized method

### DISCUSSION THREAD

**Discussion by hspringm** [cloudnative-pg/cloudnative-pg#3401] [Q&A]
API Version:  postgresql.cnpg.io/v1
 Image :     postgres:16.0

 backup:
      barmanObjectStore:
        azureCredentials:
          connectionString:
            key: AZURE_CONNECTION_STRING
            name: bkp-creds
        data:
          compression: gzip
        destinationPath: https://<>.blob.core.windows.net/<container>
        wal:
          compression: bzip2
      target: primary

--
backup job

apiVersion: postgresql.cnpg.io/v1
kind: Backup
metadata:
  name: bkp-pg
  namespace: dev-cnpg
spec:
  cluster:
    name:  dev-cnpg-cluster

After running above backup job..
Data files/folder are not created in azure blob container and copied only wal files to WAL directory. And I do see new wal files in every few hours.
 Whenever we run manual backup job, data files supposed to be copied -correct?

I see below messages in operator log file.

"error":"unrecognized method:

"msg":"Reconciler error","controller":"backup","controllerGroup":"postgresql.cnpg.io","controllerKind":"Backup","Backup":{"name":"bkp-pg-20231121000000","namespace":"dev-cnpg"},"namespace":"dev-cnpg","name":"bkp-pg-2023112100000","reconcileID":".."**,"error":"unrecognized method:** ","stacktrace":"sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\tpkg/mod/sigs.k8s.io/controller-runtime@v0.16.2/pkg/internal/controller/controller.go:329\nsigs.k8s.io/controller-runtime/pkg/internal/controller.

**Comment by gbartolini** [comment]
This is impossible for me to reproduce it. Also, please understand that as a Community it is very difficult to support on these topics. Imagine if everyone wrote to help them setup their integration with an object store.

If you think this is a bug, please file the correct issue type and avoid using discussions. Thanks. Closing this for now.
